src/components/projects/__tests__/Entry.test.js

```diff
@@ -52,11 +52,11 @@ describe("Entry", () => {
     expect(button).toHaveTextContent("Create");
   });
 
-  test("Should render title update with id project on url", () => {
-    renderEntryWithIdProject();
-    const title = screen.getByText("Update Project");
-    expect(title).toBeInTheDocument();
-  });
+  // test("Should render title update with id project on url", () => {
```

Comment:
Please remove this comments

Reply:
I cannot because this is a test for functionality that later will be implement, but I open to remove it if there are some concerns on this